lib/bloc/backup/backup_bloc.dart

```diff
@@ -251,9 +341,57 @@ class BackupBloc {
       await _breezLib.signOut();
     }
     await _breezLib.signIn(_backupServiceNeedLogin);
+    await _saveAppData();
```

Comment:
No, because _breezLib.requestBackup(); triggers a full backup afaik:
![image](https://user-images.githubusercontent.com/4012752/168120441-e4e96a69-305e-4be9-9348-626d339c6b8d.png)